src/components/PollNavbar.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+    <nav class="flex justify-between items-center bg-[#10898d] p-4">
+        <!-- Left side - Navigation Links -->
+        <div class="hidden lg:flex space-x-4 text-white">
+            <router-link to="/poll-list" active-class="bg-teal-500 text-white rounded-md shadow-lg px-4 py-2"
+                class="py-2">Polls</router-link>
+
+            <template v-if="isAdmin">
+                <router-link to="/add-poll" active-class="bg-teal-500 text-white rounded-md shadow-lg px-4 py-2"
```

Comment:
this code is repeating, create a component of nav items, and use json for rendering menu items @Shashank-002